engines/sci/graphics/screen.cpp

```diff
@@ -212,7 +192,7 @@ GfxScreen::GfxScreen(ResourceManager *resMan) : _resMan(resMan) {
 			initGraphics(_displayWidth, _displayHeight + 28 + 2, _displayWidth > 320);
 		else
 			error("Unknown SCI1.1 Mac game");
-	} else
+	} else if (getSciVersion() < SCI_VERSION_2)
 		initGraphics(_displayWidth, _displayHeight, _displayWidth > 320);
```

Comment:
this is not needed and if something goes wrong, no initGraphics() call would be done.
So better keep it like it was or add an error() in case SciVersion is => SCI_VERSION_2.